.gitignore

```diff
@@ -4,3 +4,8 @@ htmlcov/
 
 # Caches
 __pycache__/
+
+# package
+dist/
+elitech.egg-info/
```

Comment:
Shouldn't it be ` *.egg-info/`  (I don't think we want to include a EGG info lying around)